packages/client-sdk-nodejs-compression-zstd/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@gomomento/sdk-nodejs-compression-zstd",
+  "version": "0.0.1",
+  "description": "Compression extensions for Momento node.js SDK",
+  "main": "dist/src/index.js",
+  "files": [
+    "dist/src",
+    "dist/package.json"
+  ],
+  "types": "dist/src/index.d.ts",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/momentohq/client-sdk-javascript"
+  },
+  "scripts": {
+    "prebuild": "eslint . --ext .ts",
+    "test": "jest",
+    "unit-test": "jest unit",
+    "integration-test": "jest integration",
+    "build-deps": "cd ../core && npm run build && cd - && cd ../client-sdk-nodejs && npm run build && cd -",
+    "build-and-run-tests": "npm run build-deps && jest",
+    "lint": "eslint . --ext .ts",
+    "format": "eslint . --ext .ts --fix",
+    "watch": "tsc -w",
+    "build": "rm -rf dist && tsc"
+  },
+  "author": "",
```

Comment:
Momento authors

Reply:
this is the same setup we use for all the other packages for now. we can change it at some point if we want to but seems like a separate thing